src/commands/Minion/minion.ts

```diff
@@ -557,6 +557,21 @@ ${Emoji.QuestIcon} QP: ${msg.author.settings.get(UserSettings.QP)}
 			} is ${Math.floor(msg.author.maxTripLength / timeToFinish)}.`;
 		}
 
+		if (monster.consumedItem) {
+			for (const itemID of monster.consumedItem) {
```

Comment:
when you change this type to a bank you can check and remove items the same way that the create command does